### ISSUE TITLE
I want to using for training Mixed precision float16 it is possible?

### ISSUE THREAD

**Issue by search620** [daswer123/xtts-finetune-webui#39]
As for now it 

>  > Training Environment:
 | > Backend: Torch
 | > Mixed precision: False
 | > Precision: float32
 | > Current device: 0
 | > Num. of GPUs: 1
 | > Num. of CPUs: 8
 | > Num. of Torch Threads: 1
 | > Torch seed: 1
 | > Torch CUDNN: True
 | > Torch CUDNN deterministic: False
 | > Torch CUDNN benchmark: False
 | > Torch TF32 MatMul: False
